### ISSUE TITLE
doc: add usecase comment

### ISSUE THREAD

**Issue by osamuaoki** [linushdot/unsafe-mode-menu#4]
Thank you for nice extension.

This is very useful for launching javascript based functionalities.

May I suggest [one example use case as I described in Debian wiki](https://wiki.debian.org/Keyboard?highlight=%28ibus%29#How_to_create_a_custom_keyboard_shortcut_to_select_a_particular_input_source)

Inserting a long one line javascript code is quite handy to have deterministic keyboard input behavior.

```
gdbus call --session --dest org.gnome.Shell --object-path /org/gnome/Shell --method org.gnome.Shell.Eval  "imports.ui.status.keyboard.getInputSourceManager().inputSources[1].activate()"
```

**Comment by linushdot** [comment]
Thanks.

I'm not entierly sure what you are requesting here. Would you like to have the use case listed in the README file? Also, would adding a setting to enable unsafe mode by default help with your use case?

**Comment by linushdot** [comment]
Fixed in #5
